docs/integrations/github_app.md

```diff
@@ -0,0 +1,116 @@
+
+A monitored repository will automatically run a Phylum check for every commit to a Pull Request looking for changes to [supported lockfiles](https://docs.phylum.io/docs/analyzing-dependencies). If a change is found, the lockfile is submitted for analysis:
+
+![GitHub app status check in PR](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_status_check_running.png)
+
+A comment will be written to the PR if an issue is identified that fails the [defined policy](https://docs.phylum.io/docs/policy). There will be no comment if no dependencies were added or modified for a given PR. If one or more dependencies are still processing (no results available), then the note will make that clear and the CI job will only fail if dependencies that have _completed analysis results_ do not meet the active policy.
+
+### Example Comments
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - FAILED
+
+![GitHub app PR comment - failed](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_failed.png)
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - INCOMPLETE WITH FAILURE
+
+![GitHub app PR comment - incomplete with failure](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_incomplete_failure.png)
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - INCOMPLETE
+
+![GitHub app PR comment - incomplete](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_incomplete.png)
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - SUCCESS
+
+![GitHub app PR comment - success](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_success.png)
+
+---
+
+### Groups
+
+> **NOTE:** Only [Phylum PRO accounts](https://www.phylum.io/pricing) support groups.
+
+The GitHub App will automatically create a group with the name of your GitHub account/organization. All Phylum projects created by the GitHub App will be owned by that group and results can be shared by adding Phylum PRO accounts as members to the group.
+
+![Phylum groups management](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/phylum_groups_management.png)
+
+The default group that is created is owned by the account that installed the GitHub App. If you would like a different account to manage the GitHub App and group you may [transfer ownership](https://docs.phylum.io/docs/transfer_group_ownership) of the group.
+
+### Policy
+
+The Phylum GitHub app uses the [established project policy](https://docs.phylum.io/docs/policy) for making overall success/failure risk decisions. No configuration is required for setting the policy since a default policy is used for all projects. However, [Phylum PRO users](https://www.phylum.io/pricing) may specify custom policies for their projects to exercise fine-grained control over the risk decision logic.
+
+### Remediation
```

Comment:
Should we mention re-triggering the analysis once the remediations have been made?

Reply:
Yes...no problem...change coming...